service/agentconf.go

```diff
@@ -46,6 +47,10 @@ type SystemdServiceManager interface {
 
 	// StartAllAgents starts all the agents in the machine with specified series.
 	StartAllAgents(machineAgent string, unitAgents []string, dataDir string, series string) (string, []string, error)
+
+	//WriteServiceFile writes the service file in '/lib/systemd'
```

Comment:
nit -  there needs to be a space at the start of each comment line and since comments are expected to be full sentences, a full stop :D 
``` // WriteServiceFile...```

Reply:
This interface is generic across different service managers in theory, even though there's only actually an implementation for the systemd case. Rather than saying it writes the file to /lib/systemd, can you just say that it writes them to the relevant directory?